composer.json

```diff
@@ -13,12 +13,16 @@
     ],
     "require": {
         "php": ">=5.4.0",
-        "egeloen/http-adapter": "~0.8|~1.0",
-        "igorw/get-in": "~1.0"
+        "igorw/get-in": "~1.0",
+        "psr/http-message-implementation": "^1.0",
+        "php-http/client-implementation": "^1.0",
+        "php-http/message-factory": "^1.0"
```

Comment:
I think it miss the require on php-httplug, even if it's included by the client implementation, its the direct package that it is linked too.